src/views/Settings.vue

```diff
@@ -124,6 +124,31 @@
               {{ $t('settings.addStrategy') }}
             </UiButton>
           </Block>
+          <Block :title="$t('plugins')">
+            <div v-if="form.plugins">
```

Comment:
Plugins section should always be visible, otherwise if we don't have yet a plugin it would not be possible to add one

Reply:
☝️ Nevermind i can see the plugins, my bad!